#
# old_revision [4de61f8b3a8d60c481d7effd1bdf116dff3a405b]
#
# patch "libpurple/Makefile.am"
#  from [3cebc6518bf8503c7d32b0d81c76adcba7dd0ace]
#    to [076fd3ddb00c86c3ecd30d58e538bad6d324d203]
# 
============================================================
--- libpurple/Makefile.am	3cebc6518bf8503c7d32b0d81c76adcba7dd0ace
+++ libpurple/Makefile.am	076fd3ddb00c86c3ecd30d58e538bad6d324d203
@@ -233,6 +233,8 @@ libpurple_la_LIBADD = \
 	$(LIBXML_LIBS) \
 	$(LIBNM_LIBS) \
 	$(INTLLIBS) \
+	$(SASL_LIBS) \
+	$(MEANWHILE_LIBS) \
 	-lm
 
 AM_CPPFLAGS = \
